corehq/apps/cleanup/management/commands/templates/migration_attr_equality_test.j2

```diff
@@ -0,0 +1,23 @@
+from corehq.apps.cleanup.tests.util import ModelAttrEqualityHelper
+from {{models_path}} import {{class_name}}
+from {{models_path}} import SQL{{class_name}}
+
+
+class Test{{class_name}}ModelsAttrEquality(ModelAttrEqualityHelper):
+
+    def test_have_same_attrs(self):
+        couch_attrs = self.get_cleaned_couch_attrs({{class_name}})
+        sql_attrs = self.get_sql_attrs(SQL{{class_name}})
+        self.assertEqual(couch_attrs ^ sql_attrs, set())
+
+    @classmethod
+    def _couch_only_attrs(cls):
+        return {
+            # removed attrs
+            # renamed attrs
+            # not used attrs
+        }
+
+    @classmethod
+    def _sql_only_attrs(cls):
+        return {}
```

Comment:
Do these need to be methods? Would it work if they were class attributes?
```suggestion
    couch_only_attrs = {
        # removed attrs
        # renamed attrs
        # not used attrs
    }
    sql_only_attrs = {}
```
This seems like a reasonable default, and they can always be changed to methods decorated with `django.utils.functional.classproperty` if a particular migration needs to do a more complex calculation.

Also, implied in the above example, what do you think of having them not be "private" (remove leading `_`)?

Obviously these suggestions require adjustments in the code that references these attributes.

Reply:
Updated the related code and documentation as per the changes.